cloud/ecs-integration.md

```diff
@@ -0,0 +1,536 @@
+---
+title: Deploying Docker containers on ECS
+description: Deploying Docker containers on ECS
+keywords: Docker, AWS, ECS, Integration, context, Compose, cli, deploy, containers, cloud
+toc_min: 1
+toc_max: 2
+---
+
+## Overview
+
+The Docker Compose CLI enables developers to use native Docker commands to run applications in Amazon EC2 Container Service (ECS) when building cloud-native applications.
+
+The integration between Docker and Amazon ECS allows developers to use the Docker Compose CLI to:
+
+* Set up an AWS context in one Docker command, allowing you to switch from a local context to a cloud context and run applications quickly and easily
+* Simplify multi-container application development on Amazon ECS using Compose files
+
+Also see the [ECS integration architecture](ecs-architecture.md), [full list of compose features](ecs-compose-features.md) and [Compose examples for ECS integration](ecs-compose-examples.md).
+
+## Prerequisites
+
+To deploy Docker containers on ECS, you must meet the following requirements:
+
+1. Download and install Docker Desktop Stable version 2.3.0.5 or later, or Edge version 2.3.2.0 or later.
+
+    - [Download for Mac](https://desktop.docker.com/mac/edge/Docker.dmg){: target="_blank" rel="noopener" class="_"}
+    - [Download for Windows](https://desktop.docker.com/win/edge/Docker%20Desktop%20Installer.exe){: target="_blank" rel="noopener" class="_"}
+
+    Alternatively, install the [Docker Compose CLI for Linux](#install-the-docker-compose-cli-on-linux).
+
+2. Ensure you have an AWS account.
+
+Docker not only runs multi-container applications locally, but also enables
+developers to seamlessly deploy Docker containers on Amazon ECS using a
+Compose file with the `docker compose up` command. The following sections
+contain instructions on how to deploy your Compose application on Amazon ECS.
+
+## Run an application on ECS
+
+### Requirements
+
+AWS uses a fine-grained permission model, with specific role for each resource type and operation.
+
+To ensure that Docker ECS integration is allowed to manage resources for your Compose application, you have to ensure your AWS credentials grant access to following AWS IAM permissions:

```

Comment:
Is there any guide we can point them to on how to do this, for a new user this is pretty scary as a list

Reply:
fixed